native/python_package/python_driver/astimprove.py

```diff
@@ -137,10 +137,12 @@ def sync_node_pos(self, nodedict, add = 0):
             # position in that case is fine (uses the last line in that case)
             return
 
-        node_column = nodedict.get('col_offset')
-        token_startcolumn = token[TOKEN_STARTLOC][TOKENCOL]
+        token_startline = token[TOKEN_STARTLOC][TOKENROW]
+        if nodedict.get("lineno") != token_startline:
+            nodedict["lineno"] = token_startline
 
-        if node_column is None or node_column != token_startcolumn:
+        token_startcolumn = token[TOKEN_STARTLOC][TOKENCOL]
+        if nodedict.get('col_offset') != token_startcolumn:
```

Comment:
Shouldn't this be `if nodedict.get('col_offset') != token_startcolumn + add:` ?
If so, I don't see the point of checking the values, wouldn't it just be simpler with:

```python
nodedict['lineno'] = token[TOKEN_STARTLOC][TOKENROW]
nodedict['col_offset'] = token[TOKEN_STARTLOC][TOKENCOL] + add
```

Also note that there's no consistency in strins. and both single and double quoted strings are used.

Reply:
True, the "if" was a vestige of when this code was in pydetector, I've also removed the "add" parameter that was optional an unused (another vestige).